artemis-commons/src/main/java/org/apache/activemq/artemis/logs/AuditLogger.java

```diff
@@ -2639,4 +2644,18 @@ static void isAutoDelete(Object source) {
 
    @LogMessage(id = 601766, value = "User {} is getting auto-delete property on target resource: {}", level = LogMessage.Level.INFO)
    void isAutoDelete(String user, Object source);
+
+   static void createdConnection(String protocol, Object connectionID, String remoteAddress) {
```

Comment:
The difference is to disambiguate the two `createdConnection` methods from each other otherwise the code wouldn't even compile. Also, while there is a technical risk of NPE there is no practical risk because if the connection ID is `null` the code will fail before it even gets to the audit logging.